.github/workflows/schema-update.yml

```diff
@@ -0,0 +1,41 @@
+# Used to automatically create a pull request with updated schemas
+name: Automatic schema update
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
You can automate triggering based on path changes. By using a build trigger, you can remove the step of checking, then.

```
on:
  push:
    paths:
      - folder1/**
```